### ISSUE TITLE
Integrations page

### ISSUE THREAD

**Issue by olimart** [yafoy/pepito#6]
Check out `catalog` branch. Why `/integrations` not resolving ?

**Comment by adbjesus** [comment]
Can you test again after you fixed the warnings issue. It's currently working for me.

If it still doesn't work what are the errors?

**Comment by olimart** [comment]
working fine.

**Comment by olimart** [comment]
@adbjesus can you fix 'handler' definition.

This is what I wrote so far but looks inaccurate
> A Handler will dispatch the information received to the right communication channel (chat service).

Also, what does the 'ExtensionsCatalog' do again?
Required to install external extensions or something else?

**Comment by adbjesus** [comment]
So after checking out what you have in the catalog branch I believe we have different views on the plugins, you are saying that there are 3 (Adapters,Handlers,Extensions) but in my view you only have two things

* **Adapters**: Used to communicate with chat services like slack, hipchat, etc..
* **Handlers**: Used to define the commands and behavior of the bot, for example the "Info" handler allows us to use the `pepito info` command, and the "Help" handler allows us to use `pepito help` command.

In this view pepito-queenbee would be a handler because it will allow the user to have new "commands" `pepito queenbee summary [start] [end]`
Therefore I don't really see the need for an "Extensions" category since I believe we can change the behavior with just adapters and handlers. However, we need to add configuration options to the handlers, in queenbee we need to configure the token for example. And there might be other options.


##### "ExtansionsCatalog" handler
Now I call an Extension something that is either an adapter or an handler, so "Slack Adapter" is one extension, "Info Handler" is another extension, etc... They could be called Plugins as well.
With this in mind, ExtensionsCatalog is an handler that was useful when development started to see what handlers and adapters were currently activated. So you could do `pepito handlers list` and it would give the list of all active handlers. With the web app running and giving us this information it might no longer be useful. In case it stays, the name could maybe be changed to "Status" since it gives us the status of different running services. And the commands could be something like `pepito status handlers`, `pepito status adapters`.

**Comment by olimart** [comment]
> So after checking out what you have in the catalog branch I believe we have different views on the plugins, you are saying that there are 3 (Adapters,Handlers,Extensions) but in my view you only have two things

I agree. Trying to differentiate built-in stuff from external stuff anyone can download. Technically speaking, we have adapters & handlers

> In this view pepito-queenbee would be a handler

indeed.

> However, we need to add configuration options to the handlers, in queenbee we need to configure the token for example

token and base_url. That's it

> the name could maybe be changed to "Status" since it gives us the status of different running services. And the commands could be something like pepito status handlers, pepito status adapters.

Useful. Let's rename it to status and flag as default therefore can not be deactivated.
I'll change the UI accordingly but will keep the distinction between core handlers (not necessary for the bot to run but part of the default setup) and public handlers.
